js/plugins/cloud-sql-pg/test/indexer.test.ts

```diff
@@ -0,0 +1,351 @@
+import {
+  afterAll,
+  beforeAll,
+  describe,
+  expect,
+  afterEach,
+  jest,
+  test,
+} from '@jest/globals';
+import { Genkit } from 'genkit';
+import {
+  Document,
+} from 'genkit/retriever';
+import { PostgresEngine } from '../src/engine';
+import { configurePostgresIndexer } from '../src/index';
+
+import * as dotenv from 'dotenv';
+import { v4 as uuidv4 } from 'uuid';
+
+dotenv.config();
+
+const TEST_TABLE = "test_embeddings";
+const SCHEMA_NAME = "test_schema";
+
+describe("configurePostgresIndexer Integration Tests", () => {
+  let engine: PostgresEngine;
+  let mockEmbedder: any;
+  let mockAi: Genkit;
+  let testDoc: any;
+  let indexer: any;
+
+  beforeAll(async () => {
+    // Initialize PostgresEngine
+    engine = await PostgresEngine.fromEngineArgs({
+      user: process.env.DB_USER,
+      password: process.env.DB_PASSWORD,
+      host: process.env.DB_HOST,
+      database: process.env.DB_NAME,
+      port: parseInt(process.env.DB_PORT || '5432')
+    });
+
+    // Create test schema and table
+    await engine.pool.raw(`CREATE SCHEMA IF NOT EXISTS ${SCHEMA_NAME}`);
+    await engine.pool.raw(`DROP TABLE IF EXISTS ${SCHEMA_NAME}.${TEST_TABLE}`);
+
+    // Initialize the vectorstore table
+    await engine.initVectorstoreTable(TEST_TABLE, 1536, {
+      schemaName: SCHEMA_NAME,
+      contentColumn: 'content',
+      embeddingColumn: 'embedding',
+      idColumn: 'id',
+      metadataJsonColumn: 'metadata',
+      storeMetadata: true,
+      overwriteExisting: true
```

Comment:
Use custom values like in the engine tests, so we can ensure that these values are set correctly.